File: Master/1st Course/Spring semester/Moderno programmesanas valodu praktikums/Lab_8/Human.cs
namespace Lab8 {
    public class Human {
    
        public string Name { get; set; }
        public string Surname { get; set; }

        public Human() { }

        public Human(string name, string surname) {
            Name = name;
            Surname = surname;
        }

        public Human(Human human)
            : this(human.Name, human.Surname) {
        }

        public override string ToString() {
            return string.Format("Vārds: {0}, uzvārds: {1}", Name, Surname);
        }
    }
}

File: Master/1st Course/Spring semester/Moderno programmesanas valodu praktikums/Lab_1/Human.java
package lab1.concepts;

public class Human {
    protected String name;
    protected String surname;
    
    public Human() {}
    
    public Human(String name, String surname) {
        this.name = name;
        this.surname = surname;
    }
    
    public Human(Human human) {
        this (human.name, human.surname);
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSurname() {
        return surname;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    @Override
    public String toString() {
        return "Vārds: " + name + ", uzvārds: " + surname;
    }
}
